jobs/e2e_node/image-config-1-14.yaml

```diff
@@ -5,8 +5,8 @@ images:
   ubuntu:
     image: ubuntu-gke-1604-xenial-v20170420-1 # docker 1.12.6
```

Comment:
Where do we use this image in 1.14 release (noting that the docker version is unsupported and deprecated).

Reply:
no idea - cc @kubernetes/sig-node-pr-reviews

Reply:
May be will also ping sig-node slack after this goes through.

Reply:
This should be updated to a newer version, see #10820
/cc @yguo0905